test/e2e/framework/manifest/manifest.go

```diff
@@ -85,3 +91,48 @@ func StatefulSetFromManifest(fileName, ns string) (*appsv1.StatefulSet, error) {
 	}
 	return &ss, nil
 }
+
+// GetDaemonSetFromURL reads from a url and returns the daemonset in it.
+func GetDaemonSetFromURL(url string) (*appsv1.DaemonSet, error) {
```

Comment:
Why do you rename this function only?
I feel this naming way (GetFooFromBar() ) seems good for me, but this way is different from the existing functions.
That makes inconsistency at this time.
How about just moving the function without renaming now?

Reply:
see https://github.com/kubernetes/kubernetes/pull/89456#discussion_r399462716